src/main/java/com/sonyericsson/rebuild/RebuildProjectActionFactory.java

```diff
@@ -23,28 +23,34 @@
  */
 package com.sonyericsson.rebuild;
 
-import hudson.matrix.MatrixConfiguration;
 import hudson.Extension;
-import hudson.model.AbstractProject;
+import hudson.matrix.MatrixConfiguration;
 import hudson.model.Action;
-import hudson.model.TransientProjectActionFactory;
+import hudson.model.Job;
+import jenkins.model.TransientActionFactory;
 
+import javax.annotation.Nonnull;
 import java.util.Collection;
 
-import static java.util.Collections.singleton;
 import static java.util.Collections.emptyList;
+import static java.util.Collections.singleton;
 
 /**
  * Makes the rebuild button available on the project level.
  * Rebuilds the last completed build.
  */
 @Extension
-// TODO when depending on 1.548+, switch to TransientActionFactory (also for RebuildActionFactory) and take Job not AbstractProject (again consider consolidating logic with isRebuildAvailable)
```

Comment:
Should the maven manifest be updated to reflect the new dependency (so antiquated Jenkins deployments won't try and fail to use this plugin update)?

Reply:
The minimum requirement of that plugin is [2.32.2](https://github.com/jenkinsci/rebuild-plugin/blob/master/pom.xml#L13). You can also see this on the plugin page: https://plugins.jenkins.io/rebuild